### DISCUSSION TITLE
Supports overlapping on first layer

### DISCUSSION THREAD

**Discussion by Edzi11** [OrcaSlicer/OrcaSlicer#2177] [Q&A]
Im running into an issue where the supports are overlapping on the first layer which cause the print to fail. Its almost like the supports are being printed on a raft but i currently have the setting off and have even tried turning off brims as well. does the same whether it be organic supports or tree. any suggestions? 
![overlap](https://github.com/SoftFever/OrcaSlicer/assets/123626106/21f4c137-a92c-4866-bd89-8df8004ecc90)

**Comment by Steak77** [comment]
I've commented about the same behaviour on https://github.com/SoftFever/OrcaSlicer/issues/2178, which is unluckily closed. Is there any way that this could be checked?
